media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
I guess one way to avoid waiting on DOM ready might be to add a class name to `<html>`, and then just use the CSS cascade to hide/show the relevant bits? Just a thought... happy to go with whatever you think might be the most suitable/reliable method here.